fleetctl/list_units.go

```diff
@@ -90,6 +91,14 @@ Or, choose the columns to display:
 			}
 			return us.Hash
 		},
+		"uptime": func(us *schema.UnitState, full bool) string {
+			if us == nil || us.SystemdActiveState != "active" {
+				return "-"
+			}
+			tm := time.Unix(0, int64(us.SystemdActiveEnterTimestamp)*1000)
+			duration := time.Now().Sub(tm)
+			return fmt.Sprintf("%s, Since %ss", tm.Format("2006-01-02 03:04:05 PM"), strings.Split(duration.String(), ".")[0])
```

Comment:
pull `tm.Format("2006-01-02 03:04:05 PM")` into a global `var` block, and add the timezone